codecov_auth/commands/owner/interactors/save_okta_config.py

```diff
@@ -16,11 +16,11 @@ class SaveOktaConfigInput:
     url: Optional[str] = None
     enabled: bool = False
     enforced: bool = False
-    org_username: str = None
+    org_username: str | None = None
 
 
 class SaveOktaConfigInteractor(BaseInteractor):
-    def validate(self, owner: Owner):
+    def validate(self, owner: Owner) -> bool:
```

Comment:
```suggestion
    def validate(self, owner: Owner) -> None:
```
there is no `return` statement

Reply:
Whoops! Thanks for the catch!